src/components/BeerCard.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect} from 'react';
+import './BeerCard.css';
+import Nav from './Nav';
+
+const BeerCard = ({match}) => {
+  const [beer, setBeer] = useState({});
+
+  useEffect(() => {
+    fetchBeer();
+  }, []);
```

Comment:
Since you are only firing `fetchBeer()` once, you don't need to add the dependency array on line 19.